File: ed/c/examples/whatever/one.c
#include <stdio.h>

static int z = 3;
static int vals[] = {4, 5, 6};

void hello(int y)
{
  int x = 1;
  printf("hellow world: %d, %d, %d \n", x, y, z);
}

int main()
{
  hello(2);
  return 0;
}

File: ed/l/python/examples/async/one.py
import asyncio


async def cr1():
  print('hello')
  await asyncio.sleep(1)
  print('world')


def f1():
  asyncio.run(cr1())


async def cr2() -> None:
  print('hop')
  await asyncio.sleep(1)
  print('hey')


def f2():
  l = asyncio.get_event_loop()
  l.run_until_complete(cr2())
  l.close()


if __name__ == '__main__':
  # f1()
  f2()
